tpu-provisioner/admission_controller/skaffold.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: skaffold/v4beta11
```

Comment:
I haven't worked with skaffold before, is this being used to accelerate the local development process?

Reply:
Yes, you can run `skaffold dev` which will build-push-load images into a local cluster and deploy the app, then sit there and watch for changes to manifests.